sdk/src/schema.ts

```diff
@@ -25,3 +31,25 @@ export enum WrappedAZEROTx {
   Deposit = 'wrappedAZERO::deposit',
   Withdraw = 'wrappedAZERO::withdraw'
 }
+
+export class FeeTier {
```

Comment:
This will be replace soon but for now we put them here (no change required. I'm just leaving this for information)